### DISCUSSION TITLE
Searchbox Widget: Allow input type="text" instead of type="search"

### DISCUSSION THREAD

**Discussion by isjackwild** [algolia/instantsearch#4473] [General] [answered]
In [this article](https://www.algolia.com/doc/guides/building-search-ui/resources/ui-and-ux-patterns/tutorials/how-to-remove-default-browsers-behavior-on-search-input/) you describe how to remove the browsers default search input behavior, by using a searchbox with type="text" rather than type="search".

It would be really useful if the searchbox widget had the option to implement this. Currently the HTML output uses input type="search", which includes a very ugly 'X' button within the input in Chrome. This also doubles up the reset button which Algolia renders.

**Comment by francoischalifour** [comment, marked as answer]
Semantically, the `search` type makes more sense than the `text` type. If that's only a matter of browser rendering, I'd recommend adding [these CSS properties](https://stackoverflow.com/questions/9421551/how-do-i-remove-all-default-webkit-search-field-styling).

Keep in mind that [`connectSearchBox`](https://www.algolia.com/doc/api-reference/widgets/search-box/js/#connector) lets you provide your own rendering to interact with the query.

**Comment by isjackwild** [comment]
Aha perfect! I didn't realise it was possible to remove the X — I've spent ages on this before, didn't know about those psudo child elements 😅 Thanks!
